src/main/java/com/github/javafaker/Number.java

```diff
@@ -0,0 +1,67 @@
+package com.github.javafaker;
+
+import com.github.javafaker.service.RandomService;
+
+import java.math.BigDecimal;
+
+public class Number {
+
+    private final RandomService randomService;
+
+    public Number(RandomService randomService) {
+        this.randomService = randomService;
+    }
+
+    /**
+     * Returns a random number between 0 and 9
+     */
+    public int randomDigit() {
+        return randomService.nextInt(9);
+    }
+
+    /**
+     * Returns a random number between 1 and 9
+     */
+    public int randomDigitNotNull() {
```

Comment:
maybe call the method randomDigitNotZero because the method randomDigit doesn't ever return null anyway